src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java

```diff
@@ -244,7 +251,21 @@ private void fetchjButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-
         // get the auto complete suggestions based on the user input
         final List<CIValue> suggestions = new ArrayList<>();
 
+        final String elemXPath = context.computeContextXPathExpression();
+        final String attrXPath = elemXPath + "/@" + context.getAttributeName();
+        Expr attributeExpr;
+        try {
+            attributeExpr = parseXPath(attrXPath);
+        } catch (Exception e) {
+            return;
+        }
+
+
         for (final AutoComplete autoComplete : teiCompleter.getConfiguration().getAutoCompletes()) {
+            //check if attributeExpr addresses a subset of autoCompleteXPaths.attributeXPath
+            final TeiCompleter.AutoCompleteXPaths autoCompleteXPaths = teiCompleter.getXPaths(autoComplete);
+            if (!isSubset(attributeExpr, autoCompleteXPaths.getAttributeXPath())) continue;
```

Comment:
To make this less confusing could we rewrite from:

```java
if (!isSubset(attributeExpr, autoCompleteXPaths.getAttributeXPath())) continue;
```

to:
```java
if (isSubset(attributeExpr, autoCompleteXPaths.getAttributeXPath())) {
```

please?

Reply:
Done